.github/workflows/ci.yml

```diff
@@ -27,10 +29,17 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install ".[dev]"
+        python -m pip install ".[dev,spark]"
+        wget https://dlcdn.apache.org/spark/spark-3.5.4/spark-3.5.4-bin-hadoop3.tgz
+        tar -xzf spark-3.5.4-bin-hadoop3.tgz
+        export SPARK_HOME=$(pwd)/spark-3.5.4-bin-hadoop3
+        export PATH=$SPARK_HOME/sbin:$PATH
+        start-master.sh
```

Comment:
Starting a real cluster is not needed. Starting Thrift by itself will run an in-process Spark cluster.